website/content/docs/concepts/tokens.mdx

```diff
@@ -44,6 +44,21 @@ about their differences, but it is useful to understand other token concepts
 first. The features in the following sections all apply to service tokens, and
 their applicability to batch tokens is discussed later.
 
+### Token prefixes
+
+Tokens have a specific prefix that indicates their type. As of Vault 1.10, this token
+format was updated. The following table lists the prefix differences. This format
+pattern and its change also apply for recovery tokens. After the prefix, a string of
+24 or more randomly-generated characters is appended.
+
+| Token Type      | Vault 1.9.x or earlier | Vault 1.10 and later |
+|-----------------|------------------------|----------------------|
+| Service tokens  | `s.<random>`           | `hvs.<random>`       |
+| Batch tokens    | `b.<random>`           | `hvb.<random>`       |
+| Recovery tokens | `r.<random>`           | `hvr.<random>`       |
```

Comment:
The previous paragraph says there are two types of tokens, so it's a little odd that we're now listing three types. Maybe linking to the recovery token definition somewhere in here would help? https://github.com/hashicorp/vault/blob/c741eb54a3c461ad36234c765ff258589b703285/website/content/docs/concepts/recovery-mode.mdx#L49

Reply:
Great point